packages/electron/src/main/BacktraceClient.ts

```diff
@@ -37,4 +47,35 @@ export class BacktraceClient extends NodeBacktraceClient {
         this._instance = builder.build();
         return this._instance as BacktraceClient;
     }
+
+    /**
+     * Adds window information to reports as flattened attributes.
+     * @param window Window to add attributes from.
+     * @param options Options for the attribute provider.
+     *
+     * @example
+     * // Will add attributes for window under `electron.window.${myWindow.id}`
+     * client.addWindowAttributes(myWindow);
+     *
+     * // Will add attributes for window under `electron.window.myWindow`
+     * client.addWindowAttributes(myWindow, { key: 'myWindow' })
+     */
+    public addWindow(window: BrowserWindow, options?: WindowAttributeProviderOptions): this {
```

Comment:
is this useful to have? I believe it would be hard to query anything in the query builder based on the window-specific order. I believe it's worth including it in annotations however I'm not sure about attributes.

Reply:
That's why it's optional, and you can specify the `key`, so the attributes will start with `window.{key}.*`.